packages/toggle/src/Toggle.tsx

```diff
@@ -24,431 +25,416 @@ export const Mode = {
-      `;
-    })(),
-  };
-
-  const transformBySize: { [K in Size]: number } = {
-    default: 30,
-    small: 18,
-    xsmall: 12,
-  };
-
-  const sizes: { [K in Size]: number } = {
-    default: 28,
-    small: disabled ? 18 : 20,
-    xsmall: disabled ? 10 : 12,
-  };
-
-  const baseStyles = css`
-    transition: all ${transitionInMS}ms ease-in-out;
-    border-radius: 100%;
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    margin: auto;
-    overflow: hidden;
-    height: ${sizes[size]}px;
-    width: ${sizes[size]}px;
-    transform: translate3d(${checked ? transformBySize[size] : 0}px, 0, 0);
-  `;
-
-  if (disabled) {
-    return cx(
-      baseStyles,
-      css`
-        left: 1px;
-
-        &:before,
-        &:after {
-          display: none;
-        }
-      `,
-      colorSets[mode],
-    );
-  }
-
-  return cx(
-    baseStyles,
-    css`
-      left: ${size === 'default' ? 1 : 0}px;
-
-      &:before,
-      &:after {
-        content: '';
-        transition: opacity ${transitionInMS}ms ease-in-out;
-        position: absolute;
-        top: 0;
-        bottom: 0;
-        left: 0;
-        right: 0;
-      }
-
-      &:before {
-        opacity: 1;
       }
 
-      &:after {
-        opacity: 0;
-        background-image: linear-gradient(
-          rgba(220, 220, 220, 0),
-          rgba(220, 220, 220, 0.5)
-        );
+      ${sliderSelector.disabled} {
+        background-color: rgba(255, 255, 255, 0.15);
+        background-image: none;
+        box-shadow: none;
       }
     `,
-    colorSets[mode],
-  );
-};
 
-const baseLabelStyle = css`
-  transition: all ${transitionInMS}ms ease-in-out;
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  margin: auto;
-  height: 11px;
-  line-height: 11px;
-  text-transform: uppercase;
-  font-weight: bold;
-  font-size: 11px;
-  color: white;
-  user-select: none;
-`;
-
-const onLabelStyle = cx(
-  baseLabelStyle,
-  css`
-    left: 9px;
+    offLabel: css`
+      color: ${uiColors.gray.light1};
+    `,
 
-    ${toggleInput.selector}:hover ~ ${toggleGroove.selector} > &,
-    ${toggleInput.selector}:focus ~ ${toggleGroove.selector} > & {
-      color: white;
-    }
-  `,
-);
-
-const offLabelStyle = cx(
-  baseLabelStyle,
-  css`
-    right: 6px;
-    color: ${uiColors.gray.base};
-  `,
-);
-
-const getStatefulStyles = (states: StateForStyle) => ({
-  slider: getSliderStyles(states),
-  groove: getGrooveStyles(states),
-  container: getContainerStyles(states),
-});
+    onLabel: css`
+      color: ${uiColors.white};
+      text-shadow: 0 0 2px ${uiColors.blue.base};
+    `,
+  },
+} as const;
 
 interface BaseToggleProps {
+  /**
+   * Sets the size of the toggle.
+   *
+   * default: `'default'`
+   */
   size?: Size;
+
+  /**
+   * Determines if the Toggle will render the dark mode styles.
+   *
+   * default: `false`
+   */
   darkMode?: boolean;
-}
 
-type ToggleProps = BaseToggleProps &
-  Omit<HTMLElementProps<'input', never>, keyof BaseToggleProps>;
+  /**
```

Comment:
Why do we recomend this approach?

Reply:
I think you're right. Thinking more through the use-cases of toggle, it's more likely they'll want to use it as a controlled input.

Reply:
I'll update this.

Reply:
Changed it up to no longer offer that recommendation.

Reply:
👍